docs/changelog.json

```diff
@@ -2,6 +2,78 @@
   "url": "https://github.com/Shopify/hydrogen/pulls?q=is%3Apr+is%3Aclosed+%5Bci%5D+release+in%3Atitle+is%3Amerged",
   "version": "1",
   "releases": [
+    {
+      "title": "Improved sitemaps (unstable), search templates, cart gift cards, consent localization, CSP fix",
+      "version": "2024.7.5",
+      "hash": "11d4f230f3660670c08f0eff72616b354a7b12e2",
+      "pr": "https://github.com/Shopify/hydrogen/pull/2438",
+      "commit": "https://github.com/Shopify/hydrogen/commit/11d4f230f3660670c08f0eff72616b354a7b12e2",
+      "dependencies": {
+        "@remix-run/react": "^2.10.1",
+        "@remix-run/server-runtime": "^2.10.1",
+        "@shopify/hydrogen": "2024.7.4",
+        "@shopify/remix-oxygen": "^2.0.6"
+      },
+      "devDependencies": {
+        "@remix-run/dev": "^2.10.1",
+        "@shopify/cli": "3.65.3",
```

Comment:
This line will need to be updated after we release a [new version of the CLI](https://www.npmjs.com/package/@shopify/cli) that includes the current Hydrogen changes. Possibly 3.66.2